projects/ngx-favicon/src/lib/ngx-favicon.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface NgxFaviconConfig<CustomFavicon> {
```

Comment:
`CustomFavicon` should extend string.

Reply:
Oh it is actually `Dictionary<string>`

Reply:
`export interface NgxFaviconConfig<CustomFavicon extends Dictionnary<string>>`